packages/domains-table/src/point-to-wpcom-dialog/point-to-wpcom-dialog.tsx

```diff
@@ -0,0 +1,48 @@
+import { useI18n } from '@wordpress/react-i18n';
+import AcceptDialog from 'calypso/lib/accept/dialog'; // eslint-disable-line no-restricted-imports
+
+type PointToWpcomDialogProps = {
+	visible: boolean;
+	onClose: ( accepted: boolean ) => void;
+};
+
+export const PointToWpcomDialog = ( { visible, onClose }: PointToWpcomDialogProps ) => {
+	const { __ } = useI18n();
+
+	const renderContent = () => {
+		return (
+			<>
+				<p>
+					{ __(
+						'This action will update your domain settings to point to WordPress.com. Specifically, it will:'
+					) }
+				</p>
+				<ul>
+					<li>Set WordPress.com defaults nameservers</li>
```

Comment:
```suggestion
					<li>Set WordPress.com default nameservers</li>
```
Nit

Reply:
```suggestion
					<li>{ __( 'Set WordPress.com default nameservers' ) }</li>
```
Also, these strings need to be translated